spec/enterprise/controllers/enterprise/api/v1/accounts_controller_spec.rb

```diff
@@ -110,4 +110,76 @@
       end
     end
   end
+
+  describe 'GET /enterprise/api/v1/accounts/{account.id}/limits' do
+    context 'when it is an unauthenticated user' do
+      it 'returns unauthorized' do
+        get "/enterprise/api/v1/accounts/#{account.id}/limits", as: :json
+
+        expect(response).to have_http_status(:unauthorized)
+      end
+    end
+
+    context 'when it is an authenticated user' do
+      context 'when it is an agent' do
+        it 'returns unauthorized' do
+          get "/enterprise/api/v1/accounts/#{account.id}/limits",
+              headers: agent.create_new_auth_token,
+              as: :json
+
+          expect(response).to have_http_status(:unauthorized)
+        end
+      end
+
+      context 'when it is an admin' do
+        before do
+          allow_any_instance_of(Enterprise::Api::V1::AccountsController).to receive(:default_plan?).and_return(true) # rubocop:disable RSpec/AnyInstance
```

Comment:
What happens when you set real data? 
https://github.com/chatwoot/chatwoot/pull/7242/files#diff-8b5eeca673feacfb0b3b410998a95564cb059c45077f940d66435d593d2da2e9R5 
Setting Installation-Config wouldn't work in this test case.